### ISSUE TITLE
still warnings for /dev/null home

### ISSUE THREAD

**Issue by conrad784** [conrad784/docker-mediathekview-webinterface#2]
running `docker run -p 127.0.0.1:5800:5800 --rm -it -e USER_ID=99 -e GROUP_ID=99 conrad784/mediathekview-webinterface:latest` still results in lots of warnings

They seem to do no harm, but store for later fixing. 
Maybe the way to go is `usermod -d $HOME app` in the startapp script.

```org.apache.commons.configuration2.ex.ConfigurationException: Cannot create path: /dev/null/.mediathek3
	at org.apache.commons.configuration2.io.DefaultFileSystem.createPath(DefaultFileSystem.java:266)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:105)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:68)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1214)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1198)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1167)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:805)
	at mediathek.tool.ApplicationConfiguration.createDefaultConfigSettings(ApplicationConfiguration.java:135)
	at mediathek.tool.ApplicationConfiguration.loadOrCreateConfiguration(ApplicationConfiguration.java:118)
	at mediathek.tool.ApplicationConfiguration.<init>(ApplicationConfiguration.java:85)
	at mediathek.tool.ApplicationConfiguration.<clinit>(ApplicationConfiguration.java:75)
	at mediathek.Main.disableNotifications(Main.java:98)
	at mediathek.Main.main(Main.java:167)
. Startparameter: /config
. Startparameter: -M
. Portable Mode: true
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
. HikariPool-1 - Starting...
. HikariPool-1 - Start completed.
. Programmstart: 22.08.2019 09:27:12
. maxMemory: 1073 MB
. Version: MediathekView 13.3.0
. Java:
. Vendor: AdoptOpenJDK
. VMname: OpenJDK 64-Bit Server VM
. Version: 11.0.4
. Runtimeversion: 11.0.4+11
. Programmpfad: /opt/MediathekView/
. Verzeichnis Einstellungen: /config
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by mediathek.Main (file:/opt/MediathekView/MediathekView.jar) to field sun.awt.X11.XToolkit.awtAppClassName
WARNING: Please consider reporting this to the maintainers of mediathek.Main
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Can not create cache at /dev/null/.openjfx/cache/13-ea
ES2 Prism: Error - GLX extension is not supported
    GLX version 1.3 or higher is required
Can not create cache at /dev/null/.openjfx/cache/13-ea
Can not create cache at /dev/null/.openjfx/cache/13-ea
Can not create cache at /dev/null/.openjfx/cache/13-ea
. Konfig existiert nicht!
. Es gibt kein Backup
. Weder Konfig noch Backup konnte geladen werden!
org.apache.commons.configuration2.ex.ConfigurationException: Cannot create path: /dev/null/.mediathek3
	at org.apache.commons.configuration2.io.DefaultFileSystem.createPath(DefaultFileSystem.java:266)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:105)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:68)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1214)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1198)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1167)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:805)
	at mediathek.tool.ApplicationConfiguration$TimerTaskListener$WriteConfigurationTask.run(ApplicationConfiguration.java:191)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
22/08/2019 09:27:33 Got connection from client 127.0.0.1
22/08/2019 09:27:33   other clients:
22/08/2019 09:27:33 Got 'ws' WebSockets handshake
22/08/2019 09:27:33 Got protocol: binary
22/08/2019 09:27:33   - webSocketsHandshake: using binary/raw encoding
22/08/2019 09:27:33   - WebSockets client version hybi-13
22/08/2019 09:27:33 Disabled X server key autorepeat.
22/08/2019 09:27:33   to force back on run: 'xset r on' (3 times)
22/08/2019 09:27:33 incr accepted_client=1 for 127.0.0.1:39608  sock=10
22/08/2019 09:27:33 Client Protocol Version 3.8
22/08/2019 09:27:33 Protocol version sent 3.8, using 3.8
22/08/2019 09:27:33 rfbProcessClientSecurityType: executing handler for type 1
22/08/2019 09:27:33 rfbProcessClientSecurityType: returning securityResult for client rfb version >= 3.8
22/08/2019 09:27:33 copy_tiles: allocating first_line at size 41
22/08/2019 09:27:33 Pixel format for client 127.0.0.1:
22/08/2019 09:27:33   32 bpp, depth 24, little endian
22/08/2019 09:27:33   true colour: max r 255 g 255 b 255, shift r 16 g 8 b 0
22/08/2019 09:27:33 no translation needed
22/08/2019 09:27:33 Enabling NewFBSize protocol extension for client 127.0.0.1
22/08/2019 09:27:33 Enabling full-color cursor updates for client 127.0.0.1
22/08/2019 09:27:33 Using image quality level 6 for client 127.0.0.1
22/08/2019 09:27:33 Using JPEG subsampling 0, Q79 for client 127.0.0.1
22/08/2019 09:27:33 Using compression level 9 for client 127.0.0.1
22/08/2019 09:27:33 Enabling LastRect protocol extension for client 127.0.0.1
22/08/2019 09:27:33 rfbProcessClientNormalMessage: ignoring unsupported encoding type Enc(0xFFFFFECC)
22/08/2019 09:27:33 Using tight encoding for client 127.0.0.1
22/08/2019 09:27:33 client_set_net: 127.0.0.1  0.0015
22/08/2019 09:27:33 created   xdamage object: 0x40002c
. MVHttpClient: Proxy not configured
Can not create cache at /dev/null/.openjfx/cache/13-ea
Can not create cache at /dev/null/.openjfx/cache/13-ea
Can not create cache at /dev/null/.openjfx/cache/13-ea
Can not create cache at /dev/null/.openjfx/cache/13-ea
22/08/2019 09:27:39 client 1 network rate 897.9 KB/sec (129309.4 eff KB/sec)
22/08/2019 09:27:39 client 1 latency:  1.7 ms
22/08/2019 09:27:39 dt1: 0.0050, dt2: 0.0262 dt3: 0.0017 bytes: 27306
22/08/2019 09:27:39 link_rate: LR_LAN - 1 ms, 897 KB/s
. Liste Filme gelesen am: 22.08.2019, 09:27
.   erstellt am: 
.   Anzahl Filme: 0
. 
. Alte Liste erstellt am: 
.   Anzahl Filme: 0
.   Anzahl Neue: 0
. Filmliste laden (Netzwerk)
. Filme in Downloads eintragen
. -------------------------------------------------------
. Einstellungen sichern
. Einstellungen wurden gesichert
. -------------------------------------------------------
. Daten Schreiben nach: /config/mediathek.xml
. Config Schreiben nach: /config/mediathek.xml startet
. Config Schreiben beendet
. MediaDB schreiben (0) Dateien :
.    --> Start Schreiben nach: /config/mediadb.txt
.    --> geschrieben!
22/08/2019 09:27:42 created selwin: 0x40002d
22/08/2019 09:27:42 called initialize_xfixes()
org.apache.commons.configuration2.ex.ConfigurationException: Cannot create path: /dev/null/.mediathek3
	at org.apache.commons.configuration2.io.DefaultFileSystem.createPath(DefaultFileSystem.java:266)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:105)
	at org.apache.commons.configuration2.io.DefaultFileSystem.getOutputStream(DefaultFileSystem.java:68)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1214)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1198)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:1167)
	at org.apache.commons.configuration2.io.FileHandler.save(FileHandler.java:805)
	at mediathek.tool.ApplicationConfiguration$TimerTaskListener$WriteConfigurationTask.run(ApplicationConfiguration.java:191)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
. Liste Filme gelesen am: 22.08.2019, 09:27
.   erstellt am: 22.08.2019, 08:17
.   Anzahl Filme: 297297
. Liste Kompl. gelesen am: 22.08.2019, 09:27
.   Liste Kompl erstellt am: 22.08.2019, 08:17
.   Anzahl Filme: 297297
. 
. Jetzige Liste erstellt am: 22.08.2019, 08:17
.   Anzahl Filme: 297297
.   Anzahl Neue:  297297
. 
. Filme schreiben (297297 Filme) :
.    --> Start Schreiben nach: /config/filme.json
. Filme in Downloads eintragen
. Daten Schreiben nach: /config/mediathek.xml
. Config Schreiben nach: /config/mediathek.xml startet
. Config Schreiben beendet
.    --> geschrieben!
. Write duration: 3491 ms
```

**Comment by conrad784** [comment]
Calling `usermod -d $HOME app` in startapp.sh does not work, as we are already dropped our privileges.
The user `app` is created only at runtime, so this gets harder than expected. I certainly do not want to introduce sudo here.

**Comment by conrad784** [comment]
maybe fixed with latest version 13.5.0 `-DmvLogOutputPath ist nicht mehr notwendig. Die Logdatei wird nun automatisch im Portable-Modus konfiguriert.` ?
https://mediathekview.de/changelog/13-5-0/
